src/cli/serve-vite/index.js

```diff
@@ -0,0 +1,97 @@
+// External
+const importLazy = require('import-lazy')(require);
+const { createServer } = importLazy('vite');
+
+// TODO: add budle analyzer https://github.com/btd/rollup-plugin-visualizer
+// const { BundleAnalyzerPlugin } = require('webpack-bundle-analyzer');
+
+// Ours
+const { getServeConfig } = require('../../config/serve');
+const { getViteConfig } = require('../../config/vite');
+const { throws } = require('../../utils/async');
+const { cmd } = require('../../utils/color');
+const { dry, info, spin } = require('../../utils/logging');
+const { combine } = require('../../utils/structures');
+const { command } = require('../');
+const cleanCommand = require('../clean');
+const { MESSAGES } = require('./constants');
+const { getViteServerConfig } = require('../../config/viteServer');
+
+module.exports = command(
+  {
+    name: 'serve-vite',
+    nodeEnv: 'development',
+    usage: MESSAGES.usage
+  },
+  async argv => {
+    const { port } = await getServeConfig();
+    const viteConfig = getViteConfig();
```

Comment:
This `getViteConfig` abstraction looks great. It could make it really easy to just optionally add a [build.lib](https://vitejs.dev/guide/build.html#library-mode) config object for a `package` style command (similarly to how sveltekit lets you publish the `lib` folder as a library